README.md

```diff
@@ -1,18 +1,13 @@
 # SBT CodeArtifact
 
-[![Maven][maven]][mavenlink]
```

Comment:
Are we going to publish this to maven central?

Reply:
Not sure yet.

Reply:
Yes, the code for that will be in another pr.